### PR TITLE
edit docs for 0.2 release

### PR DESCRIPTION
**Pull request by Bradamant3**
Signed-off-by: JENNIFER RONDEAU <jrondeau@heptio.com>

Copyedits:
- moving toward standardized/corrected capitalization for product names, consistency, matching upstream practice (WIP)
- future tense -> present tense
- reducing duplicate content. More work to be done here.
- clarifying a few points.

### CODE REVIEW

File: discovery/README.md [hunk 1]
```diff
@@ -3,23 +3,23 @@
...
+The Kubernetes Discoverer monitors available Services and Endpoints for a single Kubernetes cluster. The credentials to access each API server are provided with a Kubernetes Secret.
 
-The Discoverer will leverage the `watch` feature of the Kubernetes API to receive changes dynamically, rather than having to poll the API. All available services & endpoints will be synchronized to the Team namespace matching the source system.
+The Discoverer leverages the watch operation of the Kubernetes API to receive changes dynamically, instead of polling the API. All available Services and Endpoints are synchronized to the Team namespace that matches the source system.
 
 ### Openstack
-The Openstack discoverer monitors all Load Balancer as a Service (LBaaS) configured as well as the corresponding Members. They are synchronized to the Team namespace as Services and Endpoints, with the Namespace being configured as the TenantName in Openstack. 
+The Openstack Discoverer monitors all configured Load Balancers as a Service (LBaaS) plus their corresponding Members. They are synchronized to the Team namespace as Services and Endpoints. The namespace ia configured as the OpenStack TenantName.
```
Comment (alexbrand) [inline discovery/README.md]: typo: "The namespace ia"

File: docs/discovery-naming-conventions.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-discover the backends and sync them to the Gimbal cluster. This is done
-by the Gimbal discovery components, such as the Kubernetes discoverer and the
-OpenStack discoverer.
+To load balance and route traffic to backend systems, Gimbal must
+discover the backends and synchronize them to the Gimbal cluster. This is done
+by the Gimbal discovery components -- currently the Kubernetes Discoverer and the
+OpenStack Discoverer.
 
 During the discovery process, Gimbal translates the discovered backends into
-Kubernetes Services and Endpoints. The name of the discovered Services and
```
Comment (alexbrand) [inline docs/discovery-naming-conventions.md]: The "Discovered Name" is a concept that gets mentioned a couple of more times in this document. Not sure if we should maintain the definition of "Discovered Name" here?

**Comment by alexbrand** [comment]
LGTM!

**Comment by rosskukulinski** [comment]
Also looks good to me!

**Pull request merged**